src/components/Table/GridTable.tsx

```diff
@@ -200,6 +200,8 @@ export interface GridTableProps<R extends Kinded, S, X> {
    */
   persistCollapse?: string;
   xss?: X;
+  /** Adds padding to a footer at the bottom of a GridTable that's rendered `as=virtual` */
+  virtualFooterPadding?: number;
```

Comment:
Hrm...yeah, this seems a little "bag on the side-ish" in terms of ergonomics of the overall props API...

Could we work this into `GridStyle` somewhere?

Would it be too magical to do:

```
const style: GridStyle = {
  rootCss: { pb: 1 },
};
```

And then in:

```
const VirtualRoot = memoizeOne
```

We could like destructure the `pb` out of `style.rootCss` and pass it to `Footer` that way?

Or is this footer bottom padding different that what `rootCss.pb` would be?

Reply:
Was chatting with @bdow a little bit, and being more concrete, my thought was that this style of prop name could end up being like:

```
${virtual || nonvirtual} x ${header || footer || something else } x ${padding || margin || something else} = ...
```

Disclaimer I'm really guessing about "just use `pb` on `rootCss`", so defer a lot on the context of whether that makes sense...

Reply:
I don't think the bottom padding would be different than what the root padding would be, we're just addressing it for a virtualized table in BP right now.

Reply:
Would destructuring `pb` our of rootCss look more like 
```
const { paddingBottom } = rootCss
```
?

Reply:
@khalidwilliams - yes, that's how'd you pull it out of the GridStyle.rootCss. And I do believe this should work, then you can apply that paddingBottom to the footer div. FWIW, since rootCss is already of type `Properties`, then I'd apply the `paddingBottom` directly... something like: 
```
const { paddingBottom } = style.rootCss || {};
...
  Footer: () => <div css={{ paddingBottom }} />,
```